src/Development/Rattle/Server.hs

```diff
@@ -33,14 +33,15 @@ import Control.Monad.IO.Class
 data RattleOptions = RattleOptions
     {rattleFiles :: FilePath -- ^ Where all my shared files go
     ,rattleSpeculate :: Maybe String -- ^ Should I speculate? Under which key?
+    ,rattleRun :: String -- ^ Key to store run# 
```

Comment:
I'm confused, what would this be? The "name" of the build system? e.g. you'd set it to "Neil's project" and then never change it again? If so, `rattleName` seems more appropriate?

Reply:
I was thinking more along the lines of whatever local machine you were running from to hopefully address the cloud issue you mentioned before.  But I am open to suggestions for a better way.

Reply:
Yeah, I think its unclear as yet, so happy to go with whatever you prefer.

Reply:
Okay, I changed the name to hopefully make the intention more clear, but maybe this will change again in the future.

Reply:
Cool. Looks like you didn't change it everywhere?

Reply:
Yes, oops.